pkg/operator/ceph/cluster/controller.go

```diff
@@ -569,6 +569,9 @@ func (c *ClusterController) onUpdate(oldObj, newObj interface{}) {
 		}
 		// remove the finalizer from the crd, which indicates to k8s that the resource can safely be deleted
 		c.removeFinalizer(newClust)
+
+		//TODO: Validate user confirmation before starting cluster cleanup
```

Comment:
With the other PR, I'm assuming we will force remove the finalizer instead of calling `handleDelete()` above.